---
title: "List of exported functions"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{Put the title of your vignette here}
  %\VignetteEngine{knitr::rmarkdown}
  \usepackage[utf8]{inputenc}
---

Tallying ChromHMM states across samples:
```
export(tally_state_summary)
export(tally_state_summary_viz)
export(tally_state_consensusprob)
```

Dealing with plotting & state color:
```
export(statecolors_makepalette)
export(plot_colorstrip)
```

Loading ChromHMM model and crudely ordering states:
```
export(chromhmm_loadmodel)
export(chromhmm_plt)
export(chromhmm_gencov)

export(chromhmm_loadenrichment)

export("%cor%")
export(chromhmm_comparestates)
export(chromhmm_comparestates_vizH)

export(chromhmm_orderstates)
export(chromhmm_makestateorderingfile)
```

## Loading

```
export(load_macs2_narrowpeak)
```

## Evaluating
Distance between sample segmentations
```
export(load_unionbedg)

export(sample_concordance)
export(sample_concordance_summary)
export(sample_concordance_viz)
export(sample_concordance_vizH)
```

Distance between state/mark decisiveness
```
export(state_decisiveness)
```

Similarity of a genomic feature between different models
```
export(regions_binmatch)
```

## Extracting
Getting epigentically marked regions:
```
export(regions_get)
```

Converting from regions to genes
```
export(regions_addgenes)
export(regions_togenes)
```

Separate column by each character
```
export(sep_colbychar)
```

Create a list of flanking & boundary regions
```
export(gr_createFlank)
export(gr_createBoundary)
```

## Transcript

Convert salmon transcript to gene levels using tximport
```
export(transcript_togene)

export(transcript_abundance)
export(transcript_abundance_viz)

export(ta_auc)

export(ta_ngenes2proportion)
export(ta_proportion2ngenes)
```

## Plotting

```
export(plot_sets)
export(plot_combinematrix)
```

```
export(makeExpression_fromLabelcol)
```
